cmake/custom/eigen.cmake

```diff
@@ -1,2 +1,2 @@
-find_package(Eigen3 3.1.0 REQUIRED)
+find_package(Eigen3 3.3 REQUIRED NO_MODULE)
```

Comment:
You can use `CONFIG` which is synonymous with `NO_MODULE` and is more canonical.

Reply:
Also, could you add a little message for the user:
```
if(TARGET Eigen3::Eigen)
  message(STATUS "Eigen3 version ${EIGEN3_VERSION_STRING} found in ${EIGEN3_INCLUDE_DIR}")
endif()
```